docs/conf.py

```diff
@@ -177,3 +178,4 @@
 }
 
 autoclass_content = 'both'
+intersphinx_mapping = {'matplotlib': ('https://matplotlib.org/stable/', None)}
```

Comment:
Can you add a scipy link too since we use some of their optimization functions in fitting?

Reply:
Done in: https://github.com/Qiskit/qiskit-experiments/pull/56/commits/1ca51a3d47f34cb8877594ae2a1b467a3d08a701